README.md

```diff
@@ -1,20 +1,34 @@
-# self-hosted-happs
+# hpos-configure-holochain
 Install self-hosted hApps from a YAML file into holochain
 
 ## Usage
 
 ```
-self-hosted-happs config.yaml
+$ hpos-configure-holochain --help
+USAGE:
+    self-hosted-happs [OPTIONS] <happ-list-path> --ui-store-folder <ui-store-folder>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+        --admin-port <admin-port>              Holochain conductor port [env: ADMIN_PORT=]  [default: 4444]
+        --happ-port <happ-port>                hApp listening port [env: HAPP_PORT=]  [default: 42233]
+        --ui-store-folder <ui-store-folder>    Path to the folder where hApp UIs will be extracted [env:
+                                               UI_STORE_FOLDER=]
+
+ARGS:
+    <happ-list-path>    Path to a YAML file containing the list of hApps to install
+
 ```
 
-where `config.yaml` is of a format `[{app_id, ui_url, dna_url}]`:
+where `happ-list-path` is of a format:
 
 ```yaml
 ---
-- app_id: elemental-chat
-  ui_url: https://s3.eu-central-1.wasabisys.com/elemetal-chat-tests/elemental-chat.zip
-  dna_url: https://s3.eu-central-1.wasabisys.com/elemetal-chat-tests/elemental-chat.dna.gz
+- installed_app_id: elemental-chat
+  version: 1
+  ui_url: https://github.com/holochain/elemental-chat-ui/releases/download/v0.0.1-alpha7/elemental-chat.zip
```

Comment:
@Br1ght0ne I've updated the README and tried running this with these updated zip & gz files as listed in the readme which are current on holo-nixpkgs, and I see this error:
```
Dec 06 13:48:05.434 DEBUG run:install_ui{happ.installed_app_id="elemental-chat"}:download_file{url=Url { scheme: "https", host: Some(Domain("github.com")), port: None, path: "/holochain/elemental-chat-ui/releases/download/v0.0.1-alpha7/elemental-chat.zip", query: None, fragment: None }}: self_hosted_happs: download successful
Dec 06 13:48:05.435  INFO run:install_ui{happ.installed_app_id="elemental-chat"}:extract_zip{unpack_path=uis/elemental-chat}: self_hosted_happs: "uis/elemental-chat", Ok(())
Dec 06 13:48:05.437  INFO run:install_ui{happ.installed_app_id="elemental-chat"}: self_hosted_happs: Err(failed to interpret file as archive
```
similarly it also is failing to interpret the gz file correctly.
Not sure what's up with that.